### ISSUE TITLE
`<wa-button>` typing doesn't allow `appearance="filled outlined"`

### ISSUE THREAD

**Issue by medmunds** [shoelace-style/webawesome#1373]
### Describe the bug
The [wa-button docs](https://webawesome.com/docs/components/button/#appearance) demonstrate `appearance="filled outlined"`, but editors that pay attention to the custom elements manifest (and TypeScript) complain that isn't a valid option for `appearance`.

### To Reproduce

1. Use an IDE with custom elements manifest support (such as recent versions of WebStorm)
2. `npm install @awesome.me/webawesome@3.0.0-beta.4` (so that Web Awesome's custom elements manifest is available to the editor) 
3. Create the following html and observe the editor's warnings/inspections:
    ```html
    <wa-button appearance="filled">Filled</wa-button>
    <wa-button appearance="filled outlined">Filled + Outlined</wa-button>
    ```

Expected: No warnings—both values of `appearance` are documented.
Actual: The second line warns that "filled outlined is not a valid value for appearance. Expected: accent, filled, outlined, or plain".

4. Create the following TypeScript and try to compile it:
    ```typescript
    import "@awesome.me/webawesome/dist/components/button/button.js";
    const button = document.createElement("wa-button");
    button.appearance = "filled outlined";
    ```

Expected: No errors
Actual: TS2322: Type '"filled outlined"' is not assignable to type '"accent" | "filled" | "outlined" | "plain"'.

### Demo

I don't know of any fiddles/codepens that pay attention to the custom elements manifest, but here's a [TypeScript playground](https://www.typescriptlang.org/play/?#code/JYWwDg9gTgLgBAIgAIEMDuBTAzhEGB0eA9JgEbra4ZEAmwWMRAxrpAHYZsxZGkCuMGBDa8BQtvgBWWBAG4AUPJZsGcfoOFwAvHBoQmfPF3xMoGFDAwBRADYYjMABQI0KALTrxCAJQLPw-BQwMHMoFDYmDG1EADNgGzsaOAgBG2AOGjl5IA). Open the Errors tab to see the problem.

### Screenshots

WebStorm:
<img width="849" height="120" alt="WebStorm warning on html from steps above" src="https://github.com/user-attachments/assets/7e704ace-f3ab-4e16-b489-84484539603e" />

TypeScript:
<img width="843" height="147" alt="TypeScript Playground error on ts from steps above" src="https://github.com/user-attachments/assets/875ce11f-383c-415b-83fc-eab073b1a862" />

### Browser / OS
- Browser/OS independent
- WA 3.0.0-beta.4

### Additional information
- I haven't investigated whether other components' appearance properties are meant to support combinations like button does.
- I've seen some discussions that suggest button *maybe* supports other combinations of appearances (e.g., "accent outlined"), but this doesn't seem to be documented anywhere.

**Comment by claviska** [comment]
Duplicate of #1127
